### DISCUSSION TITLE
libp2p Day in Bangkok is happening!

### DISCUSSION THREAD

**Discussion by dhuseby** [libp2p/cpp-libp2p#279] [Announcements]
Please join us for libp2p Day in Bangkok on Nov 12th! It's going to be another great get together of our community. Hear talks, do workshops, get answers! Register on the event page: https://t.co/tSmQgxkesi

**Comment by dhuseby** [comment]
libp2p Day @ LabWeek/DevCon/Bangkok is finalized!! The event page is here: https://lu.ma/d7kx0r2s and the event TG channel is here: https://t.me/+sb0n5HqaonJhNTVh

Here is the final schedule:

12:00 - Welcome & Intro - Dave Grantham
12:15 - A toolbox for monitoring the health of libp2p-based Peer-To-Peer Network - Dennis Trautwein
13:00 - Evaluation of the Ethereum P2P Network for Data Availability Sampling - @miga_labs
13:30 - libp2p in Ethereum: from block diffusion to PeerDAS and FullDAS - Csaba Kiraly
14:00 - Large Message Handling in GossipSub: Potential Improvements - Umar Farooq
14:30 - BREAK -
14:45 - Respawning py-libp2p : A PLDG Cohort-1 adventure - Vipul Nayyar
15:00 - Exploring how libp2p secures $8b+ for the decentralized financial system - Angus Tookey
15:15 - create-libp2p: Your P2P Network, Your Way - Chad Nehemiah
15:30 - Waku Mega Presentation Panel - Franck Royer
16:30 - libp2p Keynote - Juan Benet
17:00 - js-libp2p Custom Protocol Development Workshop - Chad Nehemiah
18:00 - Closing Goodbye

**Reply by kamilsa** [reply]
Nice agenda! Will be happy to join
